### PR TITLE
Spotify clone  project added

### PR DESCRIPTION
**Pull request by bhagwan8815**
## Related Issue
 i am add a new project in your Repository which is Spotify Clone Project.
issue no.  :  #219  

## Email id used to regsiter for VSoc'24
Email id:     ranabhagwan8815@gmail.com

## Type of PR

- [ ] Bug fix
- [ x] Feature enhancement
- [ x] Documentation update
- [ ] Security enhancement
- [ ] Other (specify): _______________

check in issue by entering [X] in boxes

## Screenshots / Videos (if applicable)

https://github.com/dhairyagothi/50_days_50_web_project/assets/130534171/7f7bcbf5-041f-4f0c-8e63-0437a62f6141

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x ] I have performed a self-review of my code.
- [x ] I have read and followed the Contribution Guidelines.
- [ x] I have tested the changes thoroughly before submitting this pull request.
- [ x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [ x] I have followed the code style guidelines of this project.
- [ x] I have checked for any existing open issues that my pull request may address.
- [ x] I have ensured that my changes do not break any existing functionality.
- [ x] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [ x] I have read the resources for guidance listed below.
- [ x] I have followed security best practices in my code changes.

check in issue by entering [X] in boxes
## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Contribution Guidelines

Thank you for considering contributing to our project! To ensure smooth collaboration and effective contribution management, please adhere to the following guidelines:

### Issue Creation

1. **Limit on Issues:**
   - Each contributor is allowed to create a maximum of **4 issues per day**. This helps us manage and address issues efficiently.

### Contribution Levels

2. **Basic Contributions:**
   - This project is primarily focused on documentation. Most of the setup has been completed, so contributors will generally need to work on basic code tasks, such as writing tests.
   - For these tasks, issues will be assigned the **Easy** label.

3. **Acknowledging Hard Work:**
   - If a contributor puts in significant effort on a task, the issue will be upgraded to **Medium**. This is our way of recognizing and appreciating extra effort.

4. **Feature Additions and Component Work:**
   - Contributors working on new features or components using JSX/TSX will be assigned a level based on the complexity and quality of their work.
   - The more complex and valuable the contribution, the higher the level assigned.

### Level Definitions

- **Easy:**
  - Tasks are straightforward, such as fixing minor bugs, writing tests, or making simple documentation updates.
- **Medium:**
  - Tasks require more effort, such as addressing complex bugs, improving existing features, or making substantial documentation improvements.
- **Hard:**
  - Tasks are highly complex and involve significant new feature development, major refactoring, or extensive contributions to the project’s core components.

We look forward to your contributions and appreciate your effort in helping us improve the project!

### CODE REVIEW

**Review by dhairyagothi — CHANGES_REQUESTED** [review]
update index.html after doing sync fork and pull

**Comment by bhagwan8815** [comment]
update file index.html pls check

**Comment by dhairyagothi** [comment]
image
@bhagwan8815 
sync fork thrn pull main branch then do edit

**Comment by bhagwan8815** [comment]
update file index.html pls check

File: public/index.html [hunk 1]
```diff
@@ -387,32 +387,13 @@
...
-            <div class="project-name">Flask Authentication App</div>
-            <a href="https://github.com/dhairyagothi/50_days_50_web_project/tree/Main/public/flask_auth_app/README.md"
-                class="demo-link">Demo</a>
-        </div>
-        <div class="table">
-            <div class="day-number">Day 67</div>
-            <div class="project-name">Blog-Website</div>
-            <a href="blog/main.html" class="demo-link">Demo</a>
-        </div>
-        <div class="table">
```
Comment (dhairyagothi) [inline public/index.html:411]: sync Fork and then edit it

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 2]
```diff
@@ -387,32 +387,13 @@
...
             <div class="day-number">Day 64</div>
-            <div class="project-name">Zomato-clone</div>
-            <a href="zomato-clone/zomato.html" class="demo-link">Demo</a>
-        </div>
-        <div class="table">
-            <div class="day-number">Day 65</div>
-            <div class="project-name">The Cube</div>
-            <a href="The Cube/index.html" class="demo-link">Demo</a>
+            <div class="project-name">Spotify_Clone_Project</div>
+            <a href="public/spotify-clone -project/index.html" class="demo-link">Demo</a>
```
Comment (dhairyagothi) [inline public/index.html]: check href public/spotify-clone -project/index.html

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Review by dhairyagothi — CHANGES_REQUESTED** [review]
before 
image

your PR
image


image

dont do like this 

sync fork then edit already told you

File: public/index.html [hunk 3]
```diff
@@ -387,32 +387,13 @@
...
             <div class="day-number">Day 64</div>
-            <div class="project-name">Zomato-clone</div>
-            <a href="zomato-clone/zomato.html" class="demo-link">Demo</a>
-        </div>
-        <div class="table">
-            <div class="day-number">Day 65</div>
-            <div class="project-name">The Cube</div>
-            <a href="The Cube/index.html" class="demo-link">Demo</a>
+            <div class="project-name">Spotify_Clone_Project</div>
+            <a href="https://github.com/bhagwan8815/Rana_repo/blob/spotify-clone/public/spotify-clone%20-project/index.html" class="demo-link">Demo</a>
```
Comment (dhairyagothi) [inline public/index.html]: add  
```html 
href="./public/spotify-clone-project/index.html" 
```

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 3 above)
Comment (bhagwan8815) [reply public/index.html]: bhai 
  href="./public/spotify-clone-project/index.html"       
  ye link add karna he kya ???

Comment (bhagwan8815) [reply public/index.html]: Bhai konsi link add karna he ye bata do pls.

Comment (dhairyagothi) [reply public/index.html]: index.html m repo ki link mat daalo ye daalo

Comment (bhagwan8815) [reply public/index.html]: Ok bhai

**Comment by dhairyagothi** [comment]
image


@bhagwan8815 dusri links remove kar di tumne

File: public/index.html [hunk 4]
```diff
@@ -387,32 +387,13 @@
...
         </div>
-        <div class="table">
-            <div class="day-number">Day 66</div>
-            <div class="project-name">Flask Authentication App</div>
-            <a href="https://github.com/dhairyagothi/50_days_50_web_project/tree/Main/public/flask_auth_app/README.md"
-                class="demo-link">Demo</a>
-        </div>
-        <div class="table">
-            <div class="day-number">Day 67</div>
```
Comment (dhairyagothi) [inline public/index.html]: day 66 67 68 missing h

**Comment by bhagwan8815** [comment]
Nahi bhai saari hi he

current 
image

yours 
image

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by bhagwan8815** [comment]
Mene Jo Repository frok ki he usme saari hi dikha raha h

**Comment by dhairyagothi** [comment]
image

@bhagwan8815  dont create multiiple branches 
do changes in main branch then push spotify clone branch m nhi h dusre days ki link

**Comment by bhagwan8815** [comment]
Bhai kar diya, pls check karo

**Review by dhairyagothi — APPROVED** [review]

**Pull request merged**